### DISCUSSION TITLE
TSC Meeting, February 2020 - TSC members please fill in topics.

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7382] [Team Posts]
Date: February, 27. 17:00 GMT
Hangout URL: https://meet.google.com/dce-dtdr-nyt
Participants:

## General
Updates from January TSC.
Opens

## Firmware
1) Support for Zephyr as SOF RTOS.
2) Refactor tracing system https://github.com/thesofproject/sof/issues/2172
3) Fundamental audio processing
    - automatic volume level
    - bass enhancement
    - sound effects (rich sound, 3d effect)
    - music demux

## Misc
1) Clarify SOF FW versioning  / release cycle VS Linux kernel release.
2) Support SOF with ML. Cadence enhanced XAF with support for ML.
3) Process for adding new TSC members

**Comment by cujomalainey** [comment]
+1 252-371-1072‬ PIN: ‪189 065‬#

**Comment by cujomalainey** [comment]
Call should connect to the hangout, call in if audio isn't working
